ucm2/sof-hda-dsp/sof-hda-dsp.conf

```diff
@@ -48,12 +61,18 @@ If.headphone {
 	]
 }
 
-If.Dmic0 {
+If.dmic {
 	Condition {
-		Type ControlExists
-		Control "name='Dmic0 Capture Volume'"
+		Type String
+		Empty "${var:DeviceDmic}"
+	}
+	False.If.Dmic0 {
```

Comment:
The logic seems OK to me. But when I test, I found "DeviceDmic" is always not defined. It seems ```True.Define.DeviceDmic "Mic1"``` doesn't take effect?

Reply:
I cannot confirm that with updated https://github.com/alsa-project/alsa-ucm-conf/pull/97/commits/214b5e6bfe75dcb829cb734ff04406bcbfb0d173 configs. Thank you for your feedback.

```
$ alsaucm alsaucm -c hw:0 dump text
...
	Device.Mic1 {
		Comment "Digital Microphone"
		Values {
			CaptureCTL "_ucm0001.hw:sofhdadsp"
			CaptureChannels 4
			CaptureMixerElem Dmic0
			CapturePCM "_ucm0001.hw:sofhdadsp,6"
			CapturePriority 100
			CaptureSwitch "Dmic0 Capture Switch"
			CaptureVolume "Dmic0 Capture Volume"
			PlaybackCTL "_ucm0001.hw:sofhdadsp"
			TQ HiFi
		}
	}
...
```